frontend/src/features/properties/components/forms/ParcelDetailForm.tsx

```diff
@@ -57,7 +60,7 @@ export const getInitialValues = (): any => {
   };
 };
 export interface IFormParcel extends IParcel {
-  financials: any;
+  financials: IFinancialYear[];
```

Comment:
format of financials has changed to accommodate react-table (which prefers a data structure with one object per table row).